docker/nginx/conf.d/client.conf

```diff
@@ -197,6 +212,31 @@ server {
 		proxy_cache_bypass $cookie_nocache $arg_nocache; # add cache bypass option
 	}
 
+	location @subdomain {
+		include /etc/nginx/conf.d/include/cors;
+
+		limit_conn downloads_by_ip 100; # ddos protection: max 100 downloads at a time
+		add_header Cache-Control "public, max-age=86400"; # allow consumer to cache response
+
+		proxy_read_timeout 600;
```

Comment:
10 minutes is a long time. Is there a reason not to use the default 60s?

Reply:
no reason to be extra tight with timeouts here - not sure why we decided originally to have it at 600s but if it aint broken don't change it :) we can always revise this later but no need to go ahead and change this in this PR